meta-lmp-base/classes/lmp.bbclass

```diff
@@ -3,7 +3,7 @@ FIO_PUSH_CMD ?= "fiopush"
 FIO_CHECK_CMD ?= "fiocheck"
 SOTA_TUF_ROOT_DIR ?= "usr/lib/sota/tuf"
 
-IMAGE_FSTYPES += "${@bb.utils.contains('EFI_PROVIDER', 'systemd-boot', 'ota-esp', ' ', d)}"
+IMAGE_FSTYPES:sota += "${@bb.utils.contains('EFI_PROVIDER', 'systemd-boot', 'ota-esp', ' ', d)}"
```

Comment:
Curious if :sota += will have bad implications here, did you check the final parsing?

Just wonder if we should instead use append here.

Reply:
Have see this on CI when testing, the += not work as you pointed. I have chnaged it to `append:sota`